test/aws/osml/model_runner/api/test_region_request.py

```diff
@@ -2,41 +2,88 @@
 
 import unittest
 
-SAMPLE_REGION_REQUEST_DATA = {
-    "tile_size": (10, 10),
-    "tile_overlap": (1, 1),
-    "tile_format": "NITF",
-    "image_id": "test-image-id",
-    "image_url": "test-image-url",
-    "region_bounds": ((0, 0), (50, 50)),
-    "model_name": "test-model-name",
-    "model_invoke_mode": "SM_ENDPOINT",
-    "output_bucket": "unit-test",
-    "output_prefix": "region-request",
-    "execution_role": "arn:aws:iam::012345678910:role/OversightMLBetaInvokeRole",
-}
+from aws.osml.model_runner.api.region_request import ModelInvokeMode, RegionRequest
+from aws.osml.model_runner.common import ImageFormats
 
 
 class TestRegionRequest(unittest.TestCase):
-    def test_invalid_data(self):
-        from aws.osml.model_runner.api.region_request import RegionRequest
+    def setUp(self):
+        """
+        Set up sample data for each test.
+        """
+        self.sample_request_data = {
+            "tile_size": (10, 10),
+            "tile_overlap": (1, 1),
+            "tile_format": "NITF",
+            "image_id": "test-image-id",
+            "image_url": "test-image-url",
+            "region_bounds": ((0, 0), (50, 50)),
+            "model_name": "test-model-name",
+            "model_invoke_mode": ModelInvokeMode.SM_ENDPOINT,
+            "execution_role": "arn:aws:iam::012345678910:role/OversightMLBetaInvokeRole",
+        }
 
-        rr = RegionRequest(
-            SAMPLE_REGION_REQUEST_DATA,
-            image_id="",
-        )
+    def tearDown(self):
+        """
+        Clean up after each test.
+        """
+        self.sample_request_data = None
 
-        assert not rr.is_valid()
+    def test_invalid_data(self):
+        """
+        Test RegionRequest with missing or invalid image_id.
+        """
+        rr = RegionRequest(self.sample_request_data, image_id="")
+        self.assertFalse(rr.is_valid())
```

Comment:
Again, looks like we are reverting pytest style assert syntax to unittest assert syntax.  I would suggest we continue to use the pytest syntax.

Reply:
Will update to pytest assertion styles